pkg/gen/python-templates/kind.py.mustache

```diff
@@ -48,6 +53,12 @@ class {{Kind}}(pulumi.CustomResource):
             __props__,
             opts)
 
+    @staticmethod
+    def get(name: str, id: Input[str], opts: Optional[ResourceOptions] = None):
+        opts_ = copy(opts) if opts is not None else ResourceOptions()
```

Comment:
Should be:   `opts = ResourceOptions(id=id) if opts is None else opts.merge(ResourceOptions(id=id)`